{{cookiecutter.github_repository}}/provisioner/vars.yml

```diff
@@ -7,6 +7,6 @@ pg_gis: {{ 'True' if cookiecutter.add_postgis.lower() == 'y' else 'False' }}
 ansible_python_interpreter: /usr/bin/python3
 django_settings: settings.production
 django_requirements_file: requirements.txt
-python_version: 3.9
+python_version: 3.11
```

Comment:
This can cause trouble with the normal ansible setup since we compile the python plugin for uWSGI (you'll find it in the ansible task).

That task will change to use 3.11 and has to be tested with uWSGI once.

Reply:
Did you test the ansible config after this change?